addons/web_editor/static/lib/odoo-editor/test/spec/copyPaste.test.js

```diff
@@ -992,52 +992,52 @@ describe('Copy and paste', () => {
             });
             it('should paste a text when selection leave a span (1)', async () => {
                 await testEditor(BasicEditor, {
-                    contentBefore: '<div>1ab<span>c[d</span>e]f</div>',
+                    contentBefore: '<div>1ab<span odoo-test-ignore-attr>c[d</span>e]f</div>',
                     stepFunction: async editor => {
                         await pasteHtml(editor, complexHtmlData);
                     },
-                    contentAfter: '<div>1ab<span>c12<i><br>ii</i>[]</span>f</div>',
+                    contentAfter: '<div>1abc12<i><br>ii[]</i>f</div>',
                 });
                 await testEditor(BasicEditor, {
-                    contentBefore: '<div>2a[b<span>c]d</span>ef</div>',
+                    contentBefore: '<div>2a[b<span odoo-test-ignore-attr>c]d</span>ef</div>',
                     stepFunction: async editor => {
                         await pasteHtml(editor, complexHtmlData);
                     },
-                    contentAfter: '<div>2a12<i><br>ii</i>[]<span>d</span>ef</div>',
+                    contentAfter: '<div>2a12<i><br>ii</i>[]def</div>',
```

Comment:
@dmo-odoo Side-effect of that normalization I mentioned, in these two similar tests in one case we end up in the `i`, in the other out of it.

Reply:
Isn't that weird ? In one case the next characters you type will have some style and in the other it won't. Should we `getDeepRange` after pasting ?

Reply:
Tradeoooooff.